climada/entity/exposures/base.py

```diff
@@ -265,7 +266,7 @@ def assign_centroids(self, hazard, method='NN', distance='haversine',
             haz_coord = hazard.centroids.coord
 
             if np.array_equal(coord, haz_coord):
-                assigned = np.arange(self.shape[0])
+                assigned = np.arange(self.gdf.shape[0])
```

Comment:
Totally unrelated with this PR, but an obvious bug that I stumbled upon occasionally.

Reply:
:D I think I already commited that fix to develop.

Reply:
I don't see it in develop: https://github.com/CLIMADA-project/climada_python/blob/develop/climada/entity/exposures/base.py#L267-L269

Reply:
Right, I added it to the latest pull request #147. Was maybe not the smartest.

Reply:
No, that's fine!